3.10/aql/functions-miscellaneous.md

```diff
@@ -446,6 +446,74 @@ guaranteed to remain the same in future versions of ArangoDB. The hash values
 should therefore be used only for temporary calculations, e.g. to compare if two
 documents are the same, or for grouping values in queries.
 
+### MINHASH()
+
+`MINHASH(values, numHashes) → hashes`
```

Comment:
This must be an array functions

Reply:
Based on the data type of the first parameter? In that case, we would need to move a lot of functions to different pages, e.g. MEDIAN(). The functions are rather grouped by "topic", and the misc function page already lists hash functions, so that seemed like the best place to me.